distribution/sender/routes/database-event-reader.xml

```diff
@@ -0,0 +1,115 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
+
+    <route id="db-event-reader">
```

Comment:
You need to set an errorHandler, should be the deadLetterChannelBuilder

Reply:
Infact we need to make sure the dead letter channel handler properly logs the event details so that we know which table row was affected and the operation

Reply:
I think when we read from DEBEZIUM_EVENT_QUEUE we should interrupt the whole process to avoid data inconsistence (because of events sequence). It's true we need to log the error so that quickly can be found and solved if this is the case.
Will you improve deadLetterChannelBuilder route??

Reply:
Yes we can stop but we need to log it